.github/workflows/ci.yml

```diff
@@ -47,19 +47,8 @@ jobs:
       - name: Build Packages
         run: pnpm run build
 
-      - name: Format Packages
-        run: pnpm run format
-
-      - name: Format package.json
-        run: pnpm run format:deps
```

Comment:
note: i removed this because we have lint:deps, which should alert us of any issues w/out the side effect of formatting deps